### PR TITLE
#1817 Resolve facebook metadata issues

### PR DESCRIPTION
**Pull request by KovalenkoStas**
Resolve https://github.com/zhukov/webogram/issues/1817
@zhukov @stek29

### CODE REVIEW

**Comment by KovalenkoStas** [comment]
Add simple description meta tag per HTML5 boilerplate - https://github.com/h5bp/html5-boilerplate/blob/master/src/index.html

**Comment by stek29** [comment]
Why have you removed site_name, and why have you reordered the tags?

**Comment by KovalenkoStas** [comment]
Wow. Sorry will fix. About reordering see my comment above.

**Comment by achirasa** [comment]
Ihave computer Com I don't i know what is it
เมื่อ 3 มี.ค. 2019 06:06 "Stanislav Kovalenko" <notifications@github.com>
เขียนว่า:

**Comment by stek29** [comment]
@KovalenkoStas what comment?

**Comment by KovalenkoStas** [comment]
@stek29 I'm added the missing `og:site_name` property. 
> why have you reordered the tags?
As I answered [here](https://github.com/zhukov/webogram/pull/1818#issuecomment-466639896).
Add simple <meta name="description"> tag near <title> per HTML5 boilerplate - https://github.com/h5bp/html5-boilerplate/blob/master/src/index.html.
It is because `<meta charset="utf-8">`,`<title></title>`, `<meta name="description" content="">` better for browser to faster get necessary meta info and better for SEO google (and other bots) get meta info quicker.

File: app/index.html [hunk 1]
```diff
@@ -33,13 +34,13 @@
   <meta name="theme-color" content="#497495">
   <meta name="google" content="notranslate">
 
+  <meta property="og:image" content="/img/og-image.jpg">
```
Comment (qWici) [inline app/index.html]: Must be "https://web.telegram.org/img/logo_share.png"

File: app/index.html [hunk 2]
```diff
@@ -33,13 +34,13 @@
...
+  <meta property="og:image:width" content="279">
+  <meta property="og:image:height" content="279">
```
Comment (qWici) [inline app/index.html]: width&height = 236

File: app/index.html [hunk 3]
```diff
@@ -7,6 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <title>Telegram Web</title>
+  <meta property="description" content="Welcome to the Web application of Telegram messenger. See https://github.com/zhukov/webogram for more info.">
```
Comment (qWici) [inline app/index.html]: Maybe:
"Official web version of Telegram - most heavily encrypted and can self-destruct messenger"

File: app/index.html (hunk 1 above)
Comment (KovalenkoStas) [reply app/index.html]: done

File: app/index.html (hunk 2 above)
Comment (KovalenkoStas) [reply app/index.html]: done

File: app/index.html (hunk 3 above)
Comment (KovalenkoStas) [reply app/index.html]: done

**Comment by zhukov** [comment]
Please revert everything except what was needed to change (image width and height) and squash changes. Especially don't change text.

**Comment by qWici** [comment]
<details>
  <summary>It's alive!</summary>
  
It's alive!
</details>

**Pull request merged**